### PR TITLE
Updating OCP-25933 for ppc64le

### PR DESCRIPTION
**Pull request by varad-ahirwadkar**
Updating test case OCP-25933 to grep correct veth interfaces.

cc: @aleskandro

### CODE REVIEW

File: features/networking/sdn.feature [hunk 1]
```diff
@@ -444,7 +444,7 @@ Feature: SDN related networking scenarios
     | nmcli device \| grep ethernet \| grep -c unmanaged |
   And evaluation of `@result[:response].split("\n")[0]` is stored in the :no_of_unmanaged_infs clipboard
   And I run commands on the host:
-    | nmcli \| grep veth \| wc -l |
+    | nmcli \| grep -v ibmveth \| grep veth \| wc -l |
```
Comment (aleskandro) [inline features/networking/sdn.feature:447]:
```suggestion
    | nmcli \| grep -P '(?<!ibm)veth' \| wc -l |
```

Comment (varad-ahirwadkar) [reply features/networking/sdn.feature]: Updated

**Comment by aleskandro** [comment]
/lgtm

@anuragthehatter are you ok with this change?

**Review by aleskandro — APPROVED** [review]

**Pull request merged**